.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+#### What does this PR do?
+#### Any background context you want to provide?
+#### What ticket does this PR close?
+Contributes to [relevant GitHub issues, eg #76]
+#### Where should the reviewer start?
+#### How should this be manually tested?
+#### Have you completed the following manual tests?
+- [ ] Manually tested [K8s CRDs](https://github.com/cyberark/secretless-broker/tree/master/test/k8s_crds)
+- [ ] Manually tested [Keychain provider](https://github.com/cyberark/secretless-broker/tree/master/test/keychain_provider)
+- [ ] Manually run the [K8s demo](https://github.com/cyberark/secretless-broker/tree/master/demos/k8s-demo)
+- [ ] Manually run [Kubernetes-Conjur demo](https://github.com/conjurdemos/kubernetes-conjur-demo) with a local Secretless Broker image build of your branch
+- [ ] Manually run the [full demo](https://github.com/cyberark/secretless-broker/tree/master/demos/full-demo) (optional)
+#### Screenshots (if appropriate)
+#### Link to build in Jenkins / GitLab (if not already visible on PR)
+#### Questions:
+> Does this work have automated integration and unit tests (if not, link to relevant GH issues to add them)?
+
+> Has this change been documented (Readme, docs, etc. - if not, link to relevant GH issues to add docs)?
+
+> Can we make a blog post, video, or animated GIF of this?
```

Comment:
I think I'd prefer to make these all `####` as well if we care about them.  They're not really related and all of the other ones are questions too.... I know we do this in our other templates but I don't think it makes sense, actually.